rmf_fleet_adapter_python/src/graph/graph.cpp

```diff
@@ -168,6 +168,10 @@ void bind_graph(py::module &m) {
       .def("get_lane", py::overload_cast<std::size_t>(&Graph::get_lane))
       .def("get_lane", py::overload_cast<std::size_t>(
           &Graph::get_lane, py::const_))
+      .def("lane_from", py::overload_cast<std::size_t, std::size_t>(
```

Comment:
Can we also add [this](https://github.com/osrf/rmf_core/blob/163d700b2a61fb4862f69840601e7e1471e39091/rmf_traffic/include/rmf_traffic/agv/Graph.hpp#L506-L507) overloaded method?
Lets call it `lanes_from_waypoint(std::size_t)`

Reply:
https://github.com/osrf/rmf_fleet_adapter_python/pull/11/commits/deb4ac601e2b8b3ec7175630c049a12180d38503